python_modules/libraries/dagster-airbyte/dagster_airbyte/asset_defs.py

```diff
@@ -708,6 +723,9 @@ def load_assets_from_airbyte_instance(
             the IOManager specified determines how the inputs to those ops are loaded. Defaults to "io_manager".
         connection_filter (Optional[Callable[[AirbyteConnectionMetadata], bool]]): Optional function which takes
             in connection metadata and returns False if the connection should be excluded from the output assets.
+        connection_to_asset_key (Optional[Callable[[AirbyteConnectionMetadat, str], AssetKey]]): Optional function which
```

Comment:
This typo occurs in other places

```suggestion
        connection_to_asset_key (Optional[Callable[[AirbyteConnectionMetadata, str], AssetKey]]): Optional function which
```